doc/CHEATSHEET.md

```diff
@@ -43,6 +43,7 @@ This document contains common usages of different resources using Fabric8 Kubern
   * [Log Options](#log-options)
   * [Serializing to yaml](#serializing-to-yaml)
   * [Running a Pod](#running-a-pod)
+  * [ClusterRole](#clusterrole)
```

Comment:
I would move this slightly upwards along with other resources(above `Top/Metrics`)